examples/units/bar_unit_demo.py

```diff
@@ -1,3 +1,13 @@
+"""
+=========================
+Group barchart with units
+=========================
+
+This is the same example as
+<a href='http://matplotlib.org/examples/api/barchart_demo.html'>
```

Comment:
I think there is probably a better way of linking between examples instead of using a hyperlink, but I can't think of how to at the moment.